src/types/users.ts

```diff
@@ -68,6 +68,10 @@ export interface UserInterface {
   user_activation_key: string;
   user_activation_forgot: boolean | null;
   // social_links: SocialLinkInterface[];
+  custom_fields?: {
+    name: string;
+    data: any;
+  }[];
```

Comment:
**suggestion(blocking):** You are using the same definition of custom fields in another place. Please define the interface separately.